### ISSUE TITLE
Keyboard navigation

### ISSUE THREAD

**Issue by blerner** [CodeGrade/hourglass#123]
We need to make sure that the UI for the exam is easily navigable by keyboard.  This involves

- [ ] making sure everything in the student navbar has a tabIndex and is focusable (we may need to change the elements to be links or buttons to ensure this)
- [ ] making sure that students can keyboard-navigate among the navbar links to different questions
- [ ] making sure that we can keyboard-navigate out of a text editor of any sort.  Since Tab and Shift+Tab are both plausibly used for indentation and formatting (in CM and Quill), let's use Ctrl+Tab instead.  This will unfortunately require us to manage the tab order ourselves, since browsers don't expose the tab order programmatically.  
- [ ] making sure that exam-authoring controls (especially the up/down/delete buttons) are keyboard accessible

**Comment by kylesferrazza** [comment]
Ctrl+Tab may not be overridable as it switches browser tabs in chrome/ff by default

**Comment by blerner** [comment]
Yeah, I just noticed that :(   But (a) we need to check whether that happens in fullscreen mode, (b) some existing accessibility tools (like VoiceOver) use it, (c) we need to check whether it's interceptable when there's only one tab in the window (and if so, we can ask students to make sure to open HG in a new window rather than new tab).
